### PR TITLE
Add dashboard errors

### PR DESCRIPTION
**Pull request by keirthana**
* Add common dashboard errors faced by users and possible actions that they could take.
    * The images were obtained from the dashboard UI team. There could be some information that needs redacting. Please advise.
    Note: Please use the asset manager URL to open and review the images. I don't see a way to review them inline.
    * Addressed all comments from the Google docs review and made the actions for troubleshooting as generic as possible.

### CODE REVIEW

**Review by morphis — APPROVED** [review]
LGTM!

File: howto/troubleshoot/dashboard-issues.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+A container can end up with an error status due to various reasons. It may not always be simple and easy to resolve this because of the variable factors involved, for example, the application that the container is hosting or any installed addons.
+
+Click on the corresponding **CONTAINER ID** and check the container details page for any possible error messages. The container details page has an **Error Message** field that could be useful.
+
+![Container details page|690x440](https://assets.ubuntu.com/v1/0ff0d3ff-container-details-error.png)
+
+The **Error Message field** can give you a starting point for identifying the issue. Some reasons for a container to go into error status could be:
+* Insufficient resources. Refer [Capacity planning](https://discourse.ubuntu.com/t/capacity-planning/28717).
+* Occasionally, access to Ubuntu archives could be a problem when creating an application. As an immediate workaround, you could disable the security update by running `amc config set container.security_updates false` or explicitly set `amc config set container.api_mirror <mirror_address>`.
```
Comment (adglkh) [inline howto/troubleshoot/dashboard-issues.md:37]: or explicitly set `amc config set container.api_mirror <mirror_address>`.
--->
 or explicitly set `amc config set container.apt_mirror <mirror_address>` to configure a container to use a different apt mirror.

File: howto/troubleshoot/dashboard-issues.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+ 
+If the reason for the container failure is not obvious from the **Error Message**, check the **Logs** tab for more information.
+
+## Logs unavailable for a container
+*Applies to: Anbox Cloud, Anbox Cloud Appliance*
+
+![Container logs unavailable|690x440](https://assets.ubuntu.com/v1/db938c41-logs-unavailable-for-container.png)
+
+Logs  are unavailable for a container when:
```
Comment (adglkh) [inline howto/troubleshoot/dashboard-issues.md:46]: Logs  are unavailable for a container when:
->
Logs are unavailable for a container when:

File: howto/troubleshoot/dashboard-issues.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+Normally, the logs are available if the container is in an error state. If the container is in the error state and yet there are no logs available, [check if you have enough resources](https://discourse.ubuntu.com/t/capacity-planning/28717).
+
+## Terminal is unavailable for a container
+*Applies to: Anbox Cloud, Anbox Cloud Appliance*
+
+![Terminal unavailable for container|690x440](https://assets.ubuntu.com/v1/e85fb9ab-terminal-unavailable-for-container.png)
+
+Terminal is not available if the container has any other status apart from **running** and **started**.  
+
+
```
Comment (adglkh) [inline howto/troubleshoot/dashboard-issues.md:59]: hm, I'm wondering why do we have multiple empty lines here?

**Review by adglkh — CHANGES_REQUESTED** [review]

File: howto/troubleshoot/dashboard-issues.md (hunk 2 above)
Comment (keirthana) [reply howto/troubleshoot/dashboard-issues.md:46]: @adglkh I don't understand this comment. Can you explain?

Comment (adglkh) [reply howto/troubleshoot/dashboard-issues.md:46]: There is a redundant space between `Logs` and `are`

Comment (keirthana) [reply howto/troubleshoot/dashboard-issues.md:46]: Ah, thank you, that was not very clear in the conversation view.

**Pull request merged**